docs/content/en/docs/getting-started/_index.md

```diff
@@ -15,58 +25,266 @@ In your local environment the following tools are required for provisioning a K8
-provisioning the cluster instances.
+The K8ssandra helm charts make installation a snap. You can override chart configurations during installation as necessary if you're an advanced user, or make changes after a default installation using `helm upgrade` at a later time.
+
+{{% alert title="Important" color="warning" %}}
+K8ssandra comes out of the box with a set of default values tailored to getting up and running quickly.  Those defaults are intended to be a great starting point for smaller-scale local development but are **not** intended for production deployments.
+{{% /alert %}}
+
+To install a single node K8ssandra cluster:
+
+1. Copy the following YAML to a file named `k8ssandra.yaml`:
+
+    ```yaml
+    cassandra:
+      version: "3.11.7"
+      clusterName: k8ssandra
+      datacenters:
+      - name: dc1
+        size: 1
+    kube-prometheus-stack:
+      grafana:
+        adminUser: admin
+        adminPassword: admin123
+    ```
 
-```console
-helm install k8ssandra k8ssandra/k8ssandra
+    The configuration file creates a K8ssandra cluster named `k8ssandra` with a datacenter, `dc1` containing a single Cassandra node.
+
+2. Use `helm install` to install K8ssandra, referring to the example configuration file:
+
+    ```bash
+    helm install -f k8ssandra.yaml k8ssandra k8ssandra/k8ssandra
+    NAME: k8ssandra
+    LAST DEPLOYED: Thu Feb 18 10:05:44 2021
+    NAMESPACE: default
+    STATUS: deployed
+    REVISION: 1
+    ```
+
+{{% alert title="Note" color="primary" %}}
+When installing K8ssandra on newer versions of Kubernetes (v1.19+), some warnings may be visible on the command line related to deprecated API usage.  This is currently a known issue and will not impact the provisioning of the cluster.
+
+```bash
+W0128 11:24:54.792095  27657 warnings.go:70] 
+apiextensions.k8s.io/v1beta1 CustomResourceDefinition is 
+deprecated in v1.16+, unavailable in v1.22+; 
+use apiextensions.k8s.io/v1 CustomResourceDefinition
 ```
 
-> When installing K8ssandra on newer versions of Kubernetes (v1.19+), some warnings may be visible on the command line 
-> related to deprecated API usage.  This is currently a known issue and will not impact the provisioning of the cluster.
-> 
-> ```
-> W0128 11:24:54.792095  27657 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
-> ```
-> 
-> For more information, check out issue [#267](https://github.com/k8ssandra/k8ssandra/issues/267).
+For more information, check out issue [#267](https://github.com/k8ssandra/k8ssandra/issues/267).
+{{% /alert %}}
+
+## Verify your K8ssandra installation
+
+Depending upon your K8 configuration, initialization of your K8ssandra installation can take a few minutes. To check the status of your K8ssandra deployment, use the `kubectl` command:
```

Comment:
Replace K8 with K8s

```suggestion
Depending upon your K8s configuration, resources available, and internet speed, initialization of your K8ssandra installation can take a few minutes. To check the status of your K8ssandra deployment, use the `kubectl` command:
```

Reply:
Do we want to bring up a rough estimate on time?

- ~2 minutes per C* node from image pull complete to boostrapped (sequential operation)
- Stargate does not start until at least 1 C* node is available
- Stargate does not start health checks for 240 seconds